subsys/bluetooth/services/fast_pair/fmdn/Kconfig

```
@@ -240,7 +240,7 @@ config BT_FAST_PAIR_FMDN_CLOCK_NVM_UPDATE_RETRY_TIME
 endif # BT_FAST_PAIR_FMDN_CLOCK
 
 config BT_FAST_PAIR_FMDN_DULT
-	bool "Enable Detecting Unwanted Location Trackers (DULT) support in FMDN"
+	bool "Detect Unwanted Location Trackers (DULT) support in FMDN"
```

```suggestion
	bool "Detecting Unwanted Location Trackers (DULT) support in FMDN"
```
See https://datatracker.ietf.org/group/dult/about/ for reference

---

Fixed